planning/obstacle_avoidance_planner/config/obstacle_avoidance_planner.param.yaml

```diff
@@ -43,10 +43,15 @@
       enable_clipping_fixed_traj: false
       non_fixed_trajectory_length: 5.0 # length of the trajectory merging optimized mpt trajectory to original(not optimized) trajectory
 
+      vehicle_stop_margin_outside_drivable_area: 1.5 # vehicle stop margin to let the vehicle stop before the calculated stop point if it is calculated outside the drivable area [m] .
+                                                     # This margin will be realized with delta_arc_length_for_mpt_points m precision.
```

Comment:
Is this comment necessary? Maybe we can remove it?

Reply:
This explanation was added based on a previous comment from @mehmetdogru . It is very beneficial to leave it for anyone would like to change it to understand it will realized by the other parameter